cmd/antrea-agent/options.go

```diff
@@ -605,6 +605,9 @@ func (o *Options) validateK8sNodeOptions() error {
 		o.dnsServerOverride = hostPort
 	}
 
+	// Ensure that the minTTL is not negative.
+	o.config.MinTTL = max(o.config.MinTTL, 0)
```

Comment:
it would be better to return an error if `o.config.MinTTL < 0` IMO

while this code is in `validateK8sNodeOptions`, I am not sure this configuration parameter is specific to the "K8sNode" case. Maybe the check should be in the parent function? cc @tnqn

Reply:
I think it can be used in ExternalNode too but guess no one validates it. I'm fine with keeping it in the parent function for now. We might create a validate function for networkpolicy related configurations later.